packages/website/src/pages/mentors.tsx

```diff
@@ -0,0 +1,11 @@
+import { MentorBenefits } from '@/components/MentorBenefits';
```

Comment:
🤔 As our website is in Polish it would be great to change the file name to `mentor`